nilearn/reporting/data/html/maskers/partials/parameters.jinja

```diff
@@ -0,0 +1,7 @@
+<section id="parameters">
+    <div class="pure-g">
+        <div class="pure-u-1 pure-u-md-3-3 table-container">
+            {{ parameters|safe }}
+        </div>
+    </div>
+</section>
```

Comment:
Are parameters listed for every estimator that generates a report, or only for maskers?

If it is a template that can be common to all estimators, it might be put under `data/html/partials`.